sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/SdkVersionUtils.cs

```diff
@@ -115,7 +115,7 @@ internal static bool IsDistro
 #elif ASP_NET_CORE_DISTRO
                 AzureMonitorAspNetCoreEventSource.Log.SdkVersionCreateFailed(ex);
 #endif
-                return null;
+                return "dotnetU:otelU:extU"; // 'U' for Unknown
```

Comment:
We never used capital words as a part of sdk string, recommend small case. Please add a detailed PR summary and TODO here stating this may not happen and we have to validate. If we see this string then what's our plan.

Reply:
`dotnetu:otelu:extu`

I have no problem with this, but I feel it's not great for readability. IMO it's difficult to differentiate the key name and the 'u' value. But since this is only for internal telemetry maybe this doesn't matter.